project/plugins.sbt

```diff
@@ -1,15 +1,12 @@
-resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
-
 // keep in sync with the play version in Dependencies
```

Comment:
Do you think this comment is still relevant? 

I noticed that the Play version here does not match the version of `play-json` in Dependencies (this was also true before this PR!)